test/Runner.java

```diff
@@ -79,7 +80,7 @@ private Path distributionTarget(File distributionFile) {
         String format = distributionFormat(distributionFile);
         return Paths.get(distributionFile.toString().replaceAll(
                 format.replace(".", "\\."), ""
-        ));
+        ) + "-" + UUID.randomUUID());
```

Comment:
This is to allow multiple runners can start at the same time.

Reply:
@lriuui0x0 Let's use the `address` as the folder name. `address` is already unique, and plus we're already printing it in the log.

It'll be useful for when we're debugging, because then we can relate which one is the directory of which server in the log easily.

Reply:
@lolski there's no `address` for `ConsoleRunner`.